elasticache/redis/Acornfile

```diff
@@ -0,0 +1,96 @@
+name:        "AWS Elasticache Redis"
+description: "Amazon's Redis compatible Elasticache cluster"
+
+args: {
+	// Name assigned to the cluster during creation. Default value is "RedisCluster".
+	clusterName: "Redis"
+
+	// Key value pairs to apply to all resources.
+	tags: {}
+
+	// Deletion protection. Must be set to false in order to delete the Redis clutser. Default value is false.
+	deletionProtection: false
+
+	// The cache node type used in the elasticache cluster. See https://aws.amazon.com/elasticache/pricing/ for a list of options. Default value is "cache.t4g.micro".
+	nodeType: "cache.t4g.micro"
+
+	// The number of cache nodes used in the elasticache cluster. Default value is 2. One primary and one read replica. Automatic failover is enabled for values >1.
+	numNodes: 2
+}
```

Comment:
I've also added the ability for the user to specify the node type and number of nodes as we discussed. We can add cluster mode args in the future.

Reply:
lets default to 1

Reply:
Set to 1 in the latest commit.